src/strongbox/ui/cli.clj

```diff
@@ -41,6 +41,16 @@
   (core/save-settings)
   nil)
 
+(defn-spec set-game-track-strictness! nil?
+  "toggles the 'strict' flag for the current addon directory and reloads addons"
+  [new-strictness-level ::sp/strict?]
+  (core/set-game-track-strictness! new-strictness-level)
+  (core/load-installed-addons)
```

Comment:
these four seem to repeat often enough. can we capture that as a semi-refresh somewhere?